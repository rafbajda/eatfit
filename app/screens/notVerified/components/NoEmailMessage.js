import React from 'react';
import { NoEmailText } from '../styles/noEmailMessageStyles';

const NoEmailMessage = () => {
    return <NoEmailText>did not receive any email?</NoEmailText>;
};

export default NoEmailMessage;
