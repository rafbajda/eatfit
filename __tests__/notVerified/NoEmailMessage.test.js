import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import NoEmailMessage from "../../app/screens/notVerified/components/NoEmailMessage";

describe('NoEmailMessage snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<NoEmailMessage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
