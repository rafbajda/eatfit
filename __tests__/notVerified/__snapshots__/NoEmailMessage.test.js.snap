// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoEmailMessage snapshot renders component 1`] = `
<Text
  className="sc-bdVaJa gHOKXO"
  style={
    Object {
      "color": "#000",
      "fontFamily": "System",
      "fontSize": 16,
    }
  }
  uppercase={false}
>
  [missing "en.info.didNotReceiveEmail" translation]
</Text>
`;
